tests/unit/ConfigLocatorTest.php

```diff
@@ -91,6 +91,24 @@ function ($item) {
         $this->assertEquals($expected, $aliasPaths);
     }
 
+    /**
+     * Test an env var supersedes user's configuration file.
+     */
+    public function testEnvVar()
+    {
+        putenv('TEST_HOME=An env overridden setting');
```

Comment:
How about making sure that we get user value initially, and that it changes after the env var is set.

Reply:
Isn't the existing test `testLoadAll()` already checking that?

Reply:
Anyway, I've updated the test.